packages/pieces/discourse/src/lib/actions/new-post.action.ts

```diff
@@ -0,0 +1,66 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import { HttpMethod, httpClient } from '@activepieces/pieces-common';
+import { discourseAuth } from '../../index';
+import { Property, createAction } from "@activepieces/pieces-framework";
+
+
+
+export const newPost = createAction({
+    auth: discourseAuth,
+    name: 'new_post',
+    description: 'Create a new post in discourse',
+    displayName: 'New Post',
```

Comment:
Can you please change to "Create Post", It's better to use verb with the action

Reply:
I would also do it for the files names and other actions as well :D

Reply:
done